x-pack/plugins/cloud_security_posture/server/lib/telemetry/collectors/indices_stats_collector.ts

```diff
@@ -69,16 +72,43 @@ const getIndexStats = async (
 
 export const getIndicesStats = async (
   esClient: ElasticsearchClient,
+  soClient: SavedObjectsClientContract,
+  coreServices: Promise<[CoreStart, CspServerPluginStartDeps, CspServerPluginStart]>,
   logger: Logger
 ): Promise<CspmIndicesStats> => {
   const [findings, latestFindings, score] = await Promise.all([
     getIndexStats(esClient, FINDINGS_INDEX_DEFAULT_NS, logger),
     getIndexStats(esClient, LATEST_FINDINGS_INDEX_DEFAULT_NS, logger),
     getIndexStats(esClient, BENCHMARK_SCORE_INDEX_DEFAULT_NS, logger),
   ]);
+
+  const [, cspServerPluginStartDeps] = await coreServices;
+
+  const cspContext = {
+    logger,
+    esClient,
+    soClient,
+    agentPolicyService: cspServerPluginStartDeps.fleet.agentPolicyService,
+    agentService: cspServerPluginStartDeps.fleet.agentService,
+    packagePolicyService: cspServerPluginStartDeps.fleet.packagePolicyService,
+    packageService: cspServerPluginStartDeps.fleet.packageService,
+    isPluginInitialized,
+  };
+
+  const status = await getCspStatus(cspContext);
+
   return {
     findings,
     latest_findings: latestFindings,
     score,
+
+    latestPackageVersion: status.latestPackageVersion,
+    cspm: status.cspm,
+    kspm: status.kspm,
+    vuln_mgmt: status.vuln_mgmt,
```

Comment:
@CohenIdo lets make sure we don't relay on status API response model and create our own mapping of telemetry data

Otherwise, if it is refactored in the future it will break the telemetry without us noticing it